### PR TITLE
Update References.tsx

### PR DESCRIPTION
**Pull request by Nicko64**
Added missing components to the references page, including SQLNinja, Tiger, Arping and CVE-2022-27925.

### CODE REVIEW

**Comment by michaeljpigott** [comment]
@Nicko64 you  need to run `npx prettier --write src/pages/References.tsx`, then add, commit and push the changes to pass the CI style check.

**Comment by Nicko64** [comment]
@michaeljpigott Fixed it so that it now passes the CI check. Had to do two commits as I accidentally left an extra space after the 1st additional commit causing it to fail the CI check again.

File: src/pages/References.tsx [hunk 1]
```diff
@@ -362,7 +362,23 @@ const ReferencesPage = () => {
                         }
                         url={"https://www.kali.org/tools/wpscan/"}
                     />
-
+                    <Reference
+                        name={"Sqlninja"}
+                        description={"Sqlninja is an SQL injection tool"}
```
Comment (michaeljpigott) [inline src/pages/References.tsx:367]: Full stop required. 
```suggestion
                        description={"Sqlninja is an SQL injection tool."}
```

File: src/pages/References.tsx [hunk 2]
```diff
@@ -362,7 +362,23 @@ const ReferencesPage = () => {
...
+                        url={"https://www.kali.org/tools/sqlninja/"}
+                    />
+                    <Reference
+                        name={"Tiger"}
+                        description={"Toolset used to perform security audits of operating system components"}
```
Comment (michaeljpigott) [inline src/pages/References.tsx:372]: Full stop required. 
```suggestion
                        description={"Toolset used to perform security audits of operating system components."}
```

File: src/pages/References.tsx [hunk 3]
```diff
@@ -362,7 +362,23 @@ const ReferencesPage = () => {
...
+                        url={"https://www.kali.org/tools/tiger/"}
+                    />
+                    <Reference
+                        name={"Arping"}
+                        description={
+                            "Arping is a tool that sends ARP/ICMP requests to a target device and displays the reply messages"
```
Comment (michaeljpigott) [inline src/pages/References.tsx:378]: Full stop required. 
```suggestion
                            "Arping is a tool that sends ARP/ICMP requests to a target device and displays the reply messages."
```

File: src/pages/References.tsx [hunk 4]
```diff
@@ -384,6 +400,11 @@ const ReferencesPage = () => {
                         description={"CVE-2022-36804 allows privilege escalation."}
                         url={"https://nvd.nist.gov/vuln/detail/CVE-2022-36804"}
                     />
+                    <Reference
+                        name={"CVE-2022-27925"}
+                        description={"Zimbra Collaboration Suite - Remote Code Execution"}
```
Comment (michaeljpigott) [inline src/pages/References.tsx:405]: This is probably a better description (taken from the user guide). 
```suggestion
                        description={"CVE-2022-27925 is a remote code execution vulnerability in the Zimbra Collaboration Suite (ZCS) that allows attackers to upload arbitrary files by abusing the mboximport functionality."}
```

**Review by michaeljpigott — CHANGES_REQUESTED** [review]
Hi @Nicko64 

Thank you for fixing the style issue. 

I recommend adding full stops to the tool descriptions and using the description of CVE-2022-27925 from the user guide. 

Can you please reorder the listing of the tools so they are in alphabetical order? 

Can you also please move CVE-2022-27925 so it is above CVE-2022-36804? 

Kind regards

Michael

File: src/pages/References.tsx (hunk 3 above)
Comment (Nicko64) [reply src/pages/References.tsx]: @michaeljpigott I've implemented the changes you recommended and made sure it passes all of the checks. It should be ready now.

**Comment by Nicko64** [comment]
@michaeljpigott The changes that you requested have been implemented.

**Review by michaeljpigott — APPROVED** [review]
Hi @Nicko64 

Thank you for making those changes. I approve this pull request. 

Kind regards

Michael

**Pull request merged**